src/lib/table/table.md

```diff
@@ -1,50 +1,185 @@
-The `mat-table` provides a Material Design styled data-table that can be used to display rows of
-data.
+The `mat-table` provides a Material Design styled data-table that can be used to display rows of 
+data. 
+
+The table's template is made up of column definitions where each definition provides the content for 
+that column's header and row cells. It also includes a `<mat-header-row>` and `<mat-row>` where each 
+takes an ordered list of the columns they should render.
+
+Passing data to the table must be done through a `DataSource`, which connects to the table by 
+providing an Observable that emits an array of data. The table picks up this data array and writes 
+a `mat-row` for each data object in the array. It is the responsibility of the `DataSource` to send 
+exactly what data should be rendered by the table, so any data manipulation features such as 
+sorting or filtering should be implemented here.
 
 This table builds on the foundation of the CDK data-table and uses a similar interface for its
 data source input and template, except that its element and attribute selectors will be prefixed 
-with `mat-` instead of `cdk-`.
-
-For more information on the interface and how it works, see the
+with `mat-` instead of `cdk-`. For detailed information on the interface and how it works, see the 
 [guide covering the CDK data-table](https://material.angular.io/guide/cdk-table).
 
+### Getting Started
+
 <!-- example(table-basic) -->
 
-### Features
+Start by writing your table's column definitions. Each column definition should be given a unique 
+name and contain the content definitions for its header and row cells.
+
+Here's a simple column definition with the name `'userName'`. The header cell contains the text 
+‘Name' and each row cell will render the `name` property of each row's data.
+
+```html
+<ng-container matColumnDef="userName">
+  <mat-header-cell *matHeaderCellDef> Name </mat-header-cell>
+  <mat-cell *matCellDef="let user"> {{user.name}} </mat-cell>
+</ng-container>
+```
+
+After you define all your columns, you'll need to provide the header and data row templates that 
+will be rendered out by the table. Each template should be given an ordered list of columns that 
+will let the table know what columns each row should render.
+
+```html
+<mat-header-row *matHeaderRowDef="[‘userName', ‘age']"></mat-header-row>
```

Comment:
Looks like only the closing ones have changed. There are a few remaining opening single-quotes that are styled.

Reply:
Ugh, thanks Google Docs